SSLsplit/scripts/autostart_sslsplit.sh

```diff
@@ -20,5 +20,5 @@ iptables -P OUTPUT ACCEPT
 sh /pineapple/modules/SSLsplit/rules/iptables
 
 iptables -t nat -A POSTROUTING -j MASQUERADE
-
+echo "If this file exists, it means that the script succesfully detected the log directory! > /pineapple/modules/SSLsplit/log/islogthere || printf "WRN: Log directory at '/pineapple/modules/SSLsplit/' was not found! We are creating it now!\n" > /pineapple/modules/SSLsplit/connections.log && mkdir /pineapple/modules/SSLsplit/log/ && echo "Directory succesfully created!" > /pineapple/modules/SSLsplit/log/islogthere
```

Comment:
"connections.log" is created by the sslsplit binary when passed as an argument ("-l logfile  connect log: log one line summary per connection to logfile", https://www.roe.ch/SSLsplit) i.e. connections.log is intended for verbose debug output from ssplit and not from the pineapple module which has its own log output (/pineapple/modules/SSLsplit/log/output_${MYTIME}.log).

If a user runs sslsplit for the first time, connections.log will not even exist and this whole check will fail, leading to even more problems.

While I do not disagree with verbosity, this particular issue pertains to a fix that ensures a directory exists and nothing else so I do not see why a user would care to know whether or not a log directory existed at runtime.